src/main/java/com/google/api/codegen/GapicContext.java

```diff
@@ -90,4 +94,17 @@ public CollectionConfig getCollectionConfig(Interface service, String entityName
     }
     return result;
   }
+
+  /**
+   * Returns a list of simple RPC methods.
+   */
+  public List<Method> getSimpleRpcMethods(Interface service) {
```

Comment:
why not just name this getMethods()? Contexts for individual languages can override it and provide all methods once they support streaming. The naming change would be slightly less intrusive that way.